shell/gpu/gpu_surface_gl_skia.cc

```diff
@@ -250,8 +249,6 @@ std::unique_ptr<SurfaceFrame> GPUSurfaceGLSkia::AcquireFrame(
       };
 
   framebuffer_info = delegate_->GLContextFramebufferInfo();
-  // Partial repaint is enabled by default
-  framebuffer_info.supports_partial_repaint = supports_partial_repaint_;
```

Comment:
If framebuffer_info.supports_partial_repaint is not set to true, partial repaint will never be triggered in the backend. That's because of this check here https://github.com/flutter/engine/blob/main/shell/common/rasterizer.cc#:~:text=if%20(frame%2D%3Eframebuffer_info().supports_partial_repaint,!layer_tree.is_leaf_layer_tracing_enabled())%20%7B.

Reply:
it should be set to true in GLContextFrameBufferInfo in the line above right?

Reply:
updated https://github.com/flutter/engine/pull/35539/files#diff-9b3fe9ec836b7a74ecb473972e17b353c302d5e49e797686292c7e6c7e19c54dR169.

Reply:
Ah yes, that's true!

Reply:
That doesn't fix things though, I'm still seeing high raster times.

Reply:
`GLContextFBOResetAfterPresent` is not called for android, so we should only use existing_damage_ if the delegate has not already set it in GLContextFrameBufferInfo